From mcuelenaere@gmail.com  Wed Sep  1 12:41:09 2010
Message-ID: <4c659d1c.487e0e0a.037b.ffffcf6a@mx.google.com>
Date: Fri, 13 Aug 2010 21:29:30 +0200
From: Maurus Cuelenaere <mcuelenaere@gmail.com>
Subject: USB: s3c-hsotg: Remove DEBUG define
To: linux-usb@vger.kernel.org, linux-samsung-soc@vger.kernel.org
Cc: ben-linux@fluff.org, dbrownell@users.sourceforge.net,
	gregkh@suse.de

DEBUG is defined unconditionally, remove it as this clutters the message log.

Signed-off-by: Maurus Cuelenaere <mcuelenaere@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/s3c-hsotg.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/usb/gadget/s3c-hsotg.c
+++ b/drivers/usb/gadget/s3c-hsotg.c
@@ -12,8 +12,6 @@
  * published by the Free Software Foundation.
 */
 
-#define DEBUG
-
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/spinlock.h>
